Source/gs/GSHandler.h

```diff
@@ -103,6 +103,11 @@ class CGSHandler
 		GS_CSR = 0x12001000,
 		GS_IMR = 0x12001010,
 		GS_SIGLBLID = 0x12001080,
+
+		// Used by funslower demo
+		// Are the 0x120010XX Addresses above correct?
+		// GS User Manual list them all under 0x120004XX.
+		GS_CSR_ALT = 0x12000400,
```

Comment:
Please place `GS_CSR_ALT` with the others above and make sure it's sorted by address. Add the comment about it being used in funslower demo on the same line. No need for the comments that asks if the ones above are correct, they are most likely are since all the games are using it.